src/compiler/crystal/syntax/parser.cr

```diff
@@ -4709,7 +4709,7 @@ module Crystal
       intel = false
       while true
         location = @token.location
-        option = parse_string_without_interpolation("asm clobber")
+        option = parse_string_without_interpolation("asm option")
```

Comment:
```crystal
asm("nop" : "in1"(foo1), "in2"(foo2)
          : "out1"(bar1), "out2"(bar2)
          : "clobber1", "clobber2"
          : "volatile", "intel")
#   each of  ^------ and ^---  is an asm option string.
```

However `"volatile, intel"` is invalid asm option string. Asm option string can be appeared multiple times with separator, but `parse_string_without_interpolation` parses only a string literal containing an asm option. I think keeping singular is better.